src/robusta/core/model/env_vars.py

```diff
@@ -54,4 +55,7 @@
 
 ROBUSTA_UI_DOMAIN = os.environ.get("ROBUSTA_UI_DOMAIN", "https://platform.robusta.dev")
 
-SENTRY_TRACES_SAMPLE_RATE = float(os.environ.get("SENTRY_TRACES_SAMPLE_RATE", 0.5))
\ No newline at end of file
+TELEMETRY_ENDPOINT = os.environ.get("TELEMETRY_ENDPOINT", "https://api.robusta.dev/telemetry")
```

Comment:
Following my change re backend profiles, the TELEMETRY_ENDPOINT should be added there as well
see https://github.com/robusta-dev/robusta/pull/270